CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ MAVEN_OPTS='--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/ja
 
 ### Running the Yarn frontend build
 
+**NOTE** If you are already use a system that has yarn installed globally, you do not need to change your path.
```

Comment:
See [this page](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) for how to display notes in GitHub-flavored Markdown. Also, isn't Yarn a proper noun that should be capitalized?